ci/images/static-checks/content/config/checkov.yaml

```diff
@@ -10,6 +10,7 @@ framework:
 skip-download: false
 output: cli
 quiet: true
-skip-check: []
+# skip HEALTHCHECK instructions check
+skip-check: [CKV_DOCKER_2]
```

Comment:
Can you try changing the formatting to:
```
skip-check:
  # skip HEATLHCHECK instructions check
  - CKV_DOCKER_2
```
That way if we need to add more checks in the future, it will be easier to identify each of them.

Reply:
Done